src/style.css

```
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-bar {
+  display: flex;
+  margin: 20px 0;
+  width: 100%;
+  height: 70px;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 4px;
+}
+
+.search-bar:last-child {
+  height: 42px;
+}
+
+.wrapper {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  cursor: pointer;
+}
+
+.search-icon--small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.search-input {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  height: 100%;
+  color: #3d4e61;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+  outline: none;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-input--small {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  padding-left: 33px;
+  outline: none;
+}
```

I recommend putting modifiers of a class next to the class itself

```suggestion
.search-input {
  border: 1px solid #e1e7ed;
  border-radius: 4px;
  width: 100%;
  height: 100%;
  color: #3d4e61;
  font-size: 16px;
  line-height: 22px;
  padding-left: 62px;
  outline: none;
}

.search-input--small {
  font-weight: 400;
  font-size: 14px;
  line-height: 19px;
  color: #3d4e61;
  padding-left: 33px;
  outline: none;
}

.search-input:hover {
  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
}

.search-input:focus {
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
  font-weight: 900;
  font-size: 16px;
  line-height: 22px;
}
```

---

done